HACKING

```diff
@@ -45,6 +48,9 @@ Unacceptable:
 Minimise your use of "goto"'s, and test every code path is reached. Also
 ensure *every* if/else, etc, even if single line, is wrapped in curly braces.
 
+if statements should always use brackets regardless of the number of lines in
```

Comment:
if => conditional (for `if` and `else`)?

Reply:
Yes, should I say if and if then here instead?

Reply:
Yes. You meant `if` and `else`, I assume.

Reply:
Indeed I did =).